nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/RunWorker.cs

```diff
@@ -122,7 +122,7 @@ private async Task<RunResult> RunForDirectory(Job job, DirectoryInfo repoContent
         _logger.Log(JsonSerializer.Serialize(discoveryResult, DiscoveryWorker.SerializerOptions));
 
         // report dependencies
-        var discoveredUpdatedDependencies = GetUpdatedDependencyListFromDiscovery(discoveryResult);
+        var discoveredUpdatedDependencies = GetUpdatedDependencyListFromDiscovery(discoveryResult, repoDirectory);
```

Comment:
Still need to verify this the correct variable to pass when running e2e.

Reply:
Resolving. Based on RunWorker unit tests repoContentsPath.FullName gives me the path I want here.